tests/integration/ceph_base_mgr_test.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	cephclient "github.com/rook/rook/pkg/daemon/ceph/client"
+	"github.com/rook/rook/tests/framework/utils"
+)
+
+type MgrCLI struct {
+	k8sh      *utils.K8sHelper
+	namespace string
+}
+
+func (cli MgrCLI) execute(command []string) (error, string) {
```

Comment:
Should this be capitalized to indicate it is a freely reusable "Public" API command?

```suggestion
func (cli MgrCLI) Execute(command []string) (error, string) {
```